worlds/AutoWorld.py

```diff
@@ -578,6 +595,20 @@ def get_data_package_data(cls) -> "GamesPackage":
         res["checksum"] = data_package_checksum(res)
         return res
 
+    @classmethod
+    def get_dynamic_data_package_data(cls):
```

Comment:
and an appropriate docstring
```suggestion
    def get_dynamic_data_package_data(cls) -> "GamesPackage | None":
```